client/src/pages/ProfileSetting/ProfileSetting.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PersonalInformationForm from './PersonalInformationForm/PersonalInformationForm';
+import useStyles from './useStyles';
+import { AppBar, Box, Grid, Tab, Tabs, Typography } from '@material-ui/core';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`scrollable-auto-tabpanel-${index}`}
+      aria-labelledby={`scrollable-auto-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: any) {
+  return {
+    id: `scrollable-auto-tab-${index}`,
+    'aria-controls': `scrollable-auto-tabpanel-${index}`,
+  };
+}
+
+export default function ProfileSetting(): JSX.Element {
```

Comment:
Isn't this feature part of this PR? https://github.com/hatchways/team-old-fashioned/pull/55/files

For the future PRs, please update the base of the PR so that only new changes are reflected

Reply:
Oops.  Sorry about that, the IDE was complaining at me.